scripts/ffgprocess.js

```diff
@@ -398,17 +398,34 @@ function processCard(card) {
 
     // Card-specific tweaks:
     //
-    // FFG treats Calibrated Laser Targeting as a Mod/Config, but we use Config/Mod
-    // FFG treats Deuterium Power Cells as Mod/Tech, but we use Tech/Mod
-    if (card.id === 549 || card.id === 654) {
-      card.upgrade_types = card.upgrade_types.reverse();
-    }
-    // Lando's Millennium Falcon [Title] doesn't properly capitalize ship name
-    if (card.id === 390) {
-      card.ability_text = card.ability_text.replace(
-        "escape craft",
-        "Escape Craft"
-      );
+    switch (card.id) {
+      case 282:
+        // card.cost.values['1'] = 4;
```

Comment:
This is commented out. Can it be removed?

Reply:
removed it